meshtastic/util.py

```diff
@@ -82,7 +82,7 @@ def fromStr(valstr):
         val = bytes()
     elif valstr.startswith("0x"):
         # if needed convert to string with asBytes.decode('utf-8')
-        val = bytes.fromhex(valstr[2:])
+        val = bytes.fromhex(valstr[2:].zfill(2))
```

Comment:
Added this because `fromStr` failed on `0x0` as fuzzed input. This is unrelated to the other changes, so I can pull this out if necessary.

Reply:
Glad to accept some test fixes alongside other stuff, so no worries there for sure!